### ISSUE TITLE
PIN/password after X time

### ISSUE THREAD

**Issue by HostFat** [ricochet-im/ricochet#336]
Can you add setting where it is possible to set a PIN and an X time?

Time example:
30 seconds
1 minutes
10 minutes
...

Then, Ricochet after the X time will ask the PIN to the user to access again to interface/chats/messages.

**Comment by ss23** [comment]
@HostFat Is this not handled by your underlying OS already? This seems like a lot of effort and work to duplicate something you already have?

**Comment by HostFat** [comment]
Yes, but sometimes you aren't on your own computer, example in a library.

**Comment by special** [comment]
Could be considered after #33
